ioctl/newcmd/account/accountnonce.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2019 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package account
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-core/ioctl"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+)
+
+// Multi-language support
+var (
+	nonceCmdShorts = map[config.Language]string{
+		config.English: "Get nonce of an account",
+		config.Chinese: "获取账户的nonce值",
+	}
+	nonceCmdUses = map[config.Language]string{
+		config.English: "nonce [ALIAS|ADDRESS]",
+		config.Chinese: "nonce [别名|地址]",
+	}
+)
+
+// NewAccountNonce represents the account nonce command
+func NewAccountNonce(client ioctl.Client) *cobra.Command {
+	use, _ := client.SelectTranslation(nonceCmdUses)
+	short, _ := client.SelectTranslation(nonceCmdShorts)
+	failToGetAddress, _ := client.SelectTranslation(failToGetAddress)
+
+	an := &cobra.Command{
+		Use:   use,
+		Short: short,
+		Args:  cobra.RangeArgs(0, 1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+			arg := ""
+			if len(args) == 1 {
```

Comment:
what is supposed to be handled if len(args) != 1?

Reply:
the number of args is none or one  by setting   Args:  cobra.RangeArgs(0, 1)